paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModelTest.kt

```diff
@@ -59,12 +58,14 @@ class USBankAccountFormViewModelTest {
             )
         ),
         isCompleteFlow = true,
-        clientSecret = PaymentIntentClientSecret("pi_12345_secret_54321"),
+        isPaymentFlow = true,
+        stripeIntentId = "id_12345",
+        clientSecret = "pi_12345_secret_54321",
+        customerId = "customer_12345",
```

Comment:
As discussed in https://stripe.slack.com/archives/C04275S9N1L/p1682967765628489?thread_ts=1682520619.353869&cid=C04275S9N1L - passing customerId will cause the endpoint to error.